CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 - Enable exemplars by default in `go.opentelemetry.io/otel/sdk/metric`. Exemplars can be disabled by setting `OTEL_METRICS_EXEMPLAR_FILTER=always_off` (#5778)
 
+### Fixed
+
+- Avoid panic in metric histograms under load. (#5815)
```

Comment:
This changelog entry is rather obscure.

```suggestion
- Fix race condition when retrieving random exemplar sample in metrics histogram. (#5815)
```

This seems still a bit obscure, so I'm open to better wordings.